src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+public class Ball {
+    private String colour;
+    private int number;
+    
+    public Ball(String colour, int number) {
+        this.colour = colour;
+        this.number = number;
+    }
+    
+    public String getColour() {
+        return this.colour;
+    }
+    
+    public int getNumber() {
+        return this.number;
+    }
+  
+    @Override
+    public String toString() {
+        StringBuilder ballString = new StringBuilder();
+      
+        ballString.append("The " + getColour() + " ball with number: " + getNumber());
```

Comment:
using StringBuilder isn't necessary here. But If you want to use it, you need to use `append` method for concatenation (ballString.append("The ").append(getColour()).....)